webservices/resources/elections.py

```diff
@@ -322,7 +322,7 @@ def filter_candidates(query, kwargs):
     query = query.filter(
         CandidateHistory.two_year_period <= kwargs['cycle'],
         CandidateHistory.two_year_period > (kwargs['cycle'] - duration),
-        CandidateHistory.election_years.any(kwargs['cycle']),
+        CandidateHistory.cycles.any(kwargs['cycle']),
```

Comment:
This causes some null district entries in some of the results, been trying to figure out how to best filter those out.